ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/cloudinit/InvertingProxyAndAgent.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.cloudbreak.ccmimpl.cloudinit;
+
+import com.cloudera.thunderhead.service.clusterconnectivitymanagementv2.ClusterConnectivityManagementV2Proto.InvertingProxy;
+import com.cloudera.thunderhead.service.clusterconnectivitymanagementv2.ClusterConnectivityManagementV2Proto.InvertingProxyAgent;
+
+class InvertingProxyAndAgent {
```

Comment:
Not a big deal but perhaps adding a `toString()` impl would be beneficial.

Reply:
added